subscriptions/reporting/subscription_reports.rst

```diff
@@ -0,0 +1,124 @@
+=================================
+Discover our subscription reports
+=================================
+
+As we know, understanding how our business is going, and where it is going, is key to success. And
+particularly so when we offer subscription services or products.
+
+Before getting to the heart of the matter, it is very important to recall certain essential concepts
+to the proper understanding of the following reports:
+
+- **Monthly Recurring Revenue (MRR)**: MRR is arguably the most important metric for subscription
+  businesses. It shows the monthly revenue earned with subscription-based products or services. It
+  is a consistent number used to track all recurring revenue over time, in monthly increments.
+
+- **Annual Run-Rate (ARR)**: ARR is the yearly version of MRR, which is based on the current MRR, to
+  estimate the coming year's performance. However, this estimation does not take variations and
+  growth into account.
+
+.. image:: media/difference-between-MRR-and-ARR.png
+  :align: center
+  :alt: Difference between MRR and ARR in Odoo Subscriptions
+
+- **Non-Recurring Revenue (NRR)**: NRR shows the revenue earned for everything else than
+  subscription-based products or services. This includes gains of a rare or unique nature that are
+  unlikely to occur in the ordinary course of businesses.
+
+- **Customer Retention**: Practices to engage existing customers to continue buying products or
+  services from your business. Customer retention can be a challenge, because you must prove you are
+  worthy of your customers' trust.
+
+- **Churn Rate**: Also known as the Rate of Attrition or Customer Churn, the churn rate can be
+  defined, in this case, as the percentage of subscribers who discontinued their subscriptions
+  within a given time period. We can distinguish two types of Churn:
+
+  - **Logo Churn**: It corresponds to the subscription cancellation rate.
+
+  - **Revenue Churn**: It corresponds to the monthly recurring revenue loss rate.
+
+  .. note::
+     Let's imagine a 2$ increase in a subscription service.
+
+     - We loss three customers out of the initial 20, which gives us a **Logo Churn** of 15%.
+     - | Therefore, the 56$ of MRR difference out of the initial 600$, gives us a **Revenue Churn**
+       | of 9,33%.
```

Comment:
```suggestion
     Let's imagine a 2$ increase in a subscription service.
     - We lost 3 customers out of the initial 20, which generates a **Logo Churn** of 15%.
     - Therefore, the 56$ of MRR difference out of the initial 600$ causes a **Revenue Churn**
        of 9,33%.
```
I am not sure, but maybe the | are not needed? To be tested (:

Reply:
Yep they are needed, I did this part with Antoine because I had a lot of trouble with it xD